transmart-core-db-tests/grails-app/conf/application.yml

```diff
@@ -1,135 +1,25 @@
+# configurations that are used only when plugin gets run (e.g. testing)
+# for grails configurations that get merged with the host application use plugin.yml
 ---
-hibernate:
-    cache:
-        queries: false
-        use_second_level_cache: true
-        use_query_cache: false
-        region.factory_class: org.hibernate.cache.ehcache.SingletonEhCacheRegionFactory
-
-dataSource:
-    pooled: true
-    jmxExport: true
-    driverClassName: org.h2.Driver
-    username: sa
-    dialect: org.hibernate.dialect.H2Dialect
-    password:
-
 environments:
-    development:
-        dataSource:
-            dbCreate: create-drop
-            logSql: false
-            url: jdbc:h2:mem:devDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE;INIT=
-              RUNSCRIPT FROM '../transmart-core-db/h2_init.sql'
     test:
-        dataSource:
-            dbCreate: create-drop
-            logSql: false
-            formatSql: true
-            url: jdbc:h2:mem:testDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE;INIT=
-              RUNSCRIPT FROM '../transmart-core-db/h2_init.sql'
-    production:
-        dataSource:
-            dbCreate: update
-            url: jdbc:h2:./prodDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE;INIT=
-              RUNSCRIPT FROM '../transmart-core-db/h2_init.sql'
-            properties:
-                jmxEnabled: true
-                initialSize: 5
-                maxActive: 50
-                minIdle: 5
-                maxIdle: 25
-                maxWait: 10000
-                maxAge: 600000
-                timeBetweenEvictionRunsMillis: 5000
-                minEvictableIdleTimeMillis: 60000
-                validationQuery: SELECT 1
-                validationQueryTimeout: 3
-                validationInterval: 15000
-                testOnBorrow: true
-                testWhileIdle: true
-                testOnReturn: false
-                jdbcInterceptors: ConnectionState
-                defaultTransactionIsolation: 2 # TRANSACTION_READ_COMMITTED
+      hibernate:
+          cache:
+              queries: false
+              use_second_level_cache: true
+              use_query_cache: false
+              region.factory_class: org.hibernate.cache.ehcache.SingletonEhCacheRegionFactory
 
----
----
-grails:
-    profile: web
-    codegen:
-        defaultPackage: transmart.core.db.tests
-    spring:
-        transactionManagement:
-            proxies: false
-info:
-    app:
-        name: '@info.app.name@'
-        version: '@info.app.version@'
-        grailsVersion: '@info.app.grailsVersion@'
-spring:
-    groovy:
-        template:
-            check-template-location: false
-# Spring Actuator Endpoints are Disabled by Default
-endpoints:
-    enabled: false
-    jmx:
-        enabled: true
+      dataSource:
+        url: jdbc:h2:mem:testDb;MVCC=TRUE;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE;INIT=RUNSCRIPT FROM '../transmart-core-db/h2_init.sql'
```

Comment:
There is a small formatting mixup in this file with a number of spaces - sometimes 2, sometimes 4

Reply:
thanks, I'll fix that.